packages/dbgpt-core/src/dbgpt/util/cli/_module.py

```
@@ -12,11 +12,16 @@
-        and hasattr(cls, "metadata")
-        and isinstance(cls.metadata, ViewMetadata)
-    )
+    try:
+        metadata = getattr(cls, "metadata", None)
+        if metadata is None:
+            return False
+        # For class-level properties or descriptors, get the actual metadata object
+        if callable(metadata):
+            metadata = metadata()
+        return issubclass(cls, BaseOperator) and isinstance(metadata, ViewMetadata)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)